CMakeLists.txt

```diff
@@ -153,6 +45,8 @@ function(pybind11_enable_warnings target_name)
   endif()
 endfunction()
 
+include(pybind11Tools)
```

Comment:
This include needs to happen before the variables are cached (line 24), otherwise it will break `add_subdirectory` in parent projects.